cognite/neat/workflows/steps/lib/graph_store.py

```diff
@@ -237,21 +239,17 @@ def run(  # type: ignore[override]
             reset_store(store_dir, graph_store.graph if graph_store else None)
             logging.info("Graph reset complete")
 
+        prefixes = rules_data.rules.prefixes if rules_data else PREFIXES
```

Comment:
Lets avoid mutating by accident the constant
```suggestion
        prefixes = rules_data.rules.prefixes if rules_data else PREFIXES.copy()
```